client/src/components/Sidebar/modules/MainMenu.tsx

```diff
@@ -65,7 +67,6 @@ interface MenuProps {
   slim: boolean;
   expandingOrCollapsing: boolean;
   currentPath: string;
-  strings: Strings;
```

Comment:
Looks like we’ll also need to remove that prop from the corresponding test cases, and Storybook.